### PR TITLE
Updates cookie-cutter to incorporate new metadata fields in the template

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- ✨ Incorporates new metadata fields from https://github.com/ITISFoundation/osparc-simcore/pull/5902
- 🔨 `.osparc` now behaves like python virtual environments, i.e.
```cmd
$ source .osparc/bin/acticate
Environment activated. To deactivate, type 'deactivate'
(.osparc) $ ooil --version
1.0.4
(.osparc) $ deactivate
osparc environment deactivated
```  

## Related issue/s

- Related to https://github.com/ITISFoundation/osparc-simcore/pull/5902
- Part of ITISFoundation/osparc-issues#1404 

## How to test

 Manual testing:
 - install cc 
```
pip install cookiecutter jinja2_time
```
 - run cc against this dev branch and answer all prompts
```
cookiecutter gh:pcrespov/cookiecutter-osparc-service -c is1404/new-version_display
```
- cd your project folder
```
source .osparc/bin/acticate
ooil --version
ooil build
yq e . docker-compose.yml
```
image

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]

File: {{cookiecutter.project_slug}}/.osparc/bin/activate [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Store the current PATH
+export _OLD_OSPARC_PATH=$PATH
```
Comment (sanderegg) [inline {{cookiecutter.project_slug}}/.osparc/bin/activate:4]: waow I was very worried you really called it ```acticate```...

**Review by sanderegg — APPROVED** [review]
nice actication... ;)

**Review by elisabettai — APPROVED** [review]
Looks great!

As discussed yesterday, I agree that having a postprocessor that takes care of missing `version-display` will be useful. I imagine most users of the cookiecutter don't want to specify 2 different versions.

Also a question: have you also set up "recipes" to update `version` and `version-display` separately? Or there's no need?

**Comment by pcrespov** [comment]
> Also a question: have you also set up "recipes" to update `version` and `version-display` separately? Or there's no need?

@elisabettai @mguidon An easy what to go is to add `version_display` in `.cookiecutterrc` and then `make replay`

**Pull request merged**